tox.ini

```diff
@@ -24,24 +24,23 @@ passenv =
 [testenv:fmt]
 description = Apply coding style standards to code
 deps =
-    black
     ruff
 commands =
-    black {[vars]all_path}
     ruff check --fix {[vars]all_path}
+    ruff format {[vars]all_path}
 
 [testenv:lint]
 description = Check code against coding style standards
 deps =
-    black
     ruff
     codespell
 commands =
     # uncomment the following line if this charm owns a lib
     # codespell {[vars]lib_path}
-    codespell .
+    # codespell has issues, it's trying to lint pyproject.toml, but only in CI
```

Comment:
Have we figured this out?

Reply:
We have not yet; I'll open an issue to make sure we don't lose track of this